hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/utils/TextFileBasedIdentityHandler.java

```diff
@@ -17,6 +17,7 @@
  */
 package org.apache.hadoop.fs.azurebfs.utils;
 
+import org.apache.hadoop.io.IOUtils;
```

Comment:
nit: can you move down to the rest of the "real" org.apache.hadoop imports. 

This matters during backporting, as the `org.apache.hadoop.thirdparty.` prefix gets cut when we go back to unshaded guava. It's becoming a lot harder to manage these imports now

Reply:
moved import